src/transformState.js

```diff
@@ -61,8 +61,28 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    switch (transform.operation) {
+      case 'addProperties':
+        Object.assign(state, transform.properties);
+        break;
+      case 'removeProperties':
+        for (const value of transform.properties) {
+          delete state[value];
+        }
+        break;
+
+      case 'clear':
+
+        for (const item of Object.keys(state)) {
```

Comment:
don't use Object.keys here, let's use 'for in' loop to iterate through state's keys